jubilant/__init__.py

```diff
@@ -41,4 +41,4 @@
     'temp_model',
 ]
 
-__version__ = '1.0.0b3'
+__version__ = '0.1.0'
```

Comment:
I was going to ship actual pre-release beta versions (and already shipped `v1.0.0b1` and `v1.0.0b3`), but earlier I had published `0.0.0` with nothing in it just to claim the name `jubilant`. So when people type `pip install jubilant` they get the useless `0.0.0` version.

So I think it's better to use 0.1.0 "real" versions leading up to 1.0.0 instead of pre-release "b" versions.